### ISSUE TITLE
Aux window: fails to open editor (web)

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#199836]
In web opening an aux window results in an exception.

**Comment by justschen** [comment]
Works fine for me now - only thing is that new copied windows don't have codicons.

![Screenshot 2023-12-04 at 1 14 50 PM](https://github.com/microsoft/vscode/assets/54879025/2b9320e1-b85d-4e3b-9506-7d21b08261a8)

Also, was testing behavior vs. what we have on desktop in insiders, and noticed that on web we cannot copy a copied window, but not sure if this is intended or if it's even needed in real life scenarios.
![Screenshot 2023-12-04 at 1 16 04 PM](https://github.com/microsoft/vscode/assets/54879025/5e7ad259-ab74-46a3-b15e-8d23302e89d2)

**Comment by bpasero** [comment]
Yeah you have to disable popup blockers in web. I can see if we get the icon to work.
